cudamapper/tests/Test_CudamapperMinimizer.cpp

```diff
@@ -26,19 +26,21 @@ void test_function(const std::uint64_t number_of_reads_to_add,
                    const std::vector<Minimizer::ReadidPositionDirection>& expected_rest_h,
                    const bool hash_minimizers)
 {
-    device_buffer<char> merged_basepairs_d(merged_basepairs_h.size());
+    std::shared_ptr<DeviceAllocator> allocator(new CudaMallocAllocator());
```

Comment:
See https://www.youtube.com/watch?v=xGDLkt-jBJ4&t=28m23s for more details

Reply:
Thanks @mimaric and @ahehn-nv for the insight into this. The change for using `make_shared` instead of `new` is already in place for all the Test_* files.